search/retrieval.cpp

```diff
@@ -464,7 +465,7 @@ bool Retrieval::InitBucketStrategy(Bucket & bucket, double scale)
 
   try
   {
-    auto collector = [&](trie::ValueReader::ValueType const & value)
+    auto collector = [&](FeatureIndexValue const & value)
```

Comment:
Retrieval was fixed, please, take a look a to https://github.com/mapsme/omim/pull/349